scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -382,10 +382,38 @@
                     }
                 },
                 "username": "cool-james",
-                "roles": [
```

Comment:
Not sure why the formatting is coming through differently for you. I'm not too worried about it, but if I see this again, we really need to all get on the same page with Prettier. The only diff here should be the newly added test user.